src/components/ButtonPanel.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+export default function ButtonPanel({ children }) {
+  return <div className="button_panel">{children}</div>;
```

Comment:
Hint:

Microverse encourage DRY-(Don't Repeat Yourself) code, In your case; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```